src/common.speech/ConversationServiceRecognizer.ts

```diff
@@ -113,49 +111,44 @@ export class ConversationServiceRecognizer extends ServiceRecognizerBase {
                 EnumTranslation.implTranslateErrorDetails(cancellationErrorCode));
 
         } else {
-            if (!(this.privRequestSession.isSpeechEnded && resultReason === ResultReason.NoMatch && simple.RecognitionStatus !== RecognitionStatus.InitialSilenceTimeout)) {
+            if (simple.RecognitionStatus !== RecognitionStatus.EndOfDictation) {
                 if (this.privRecognizerConfig.parameters.getProperty(OutputFormatPropertyName) === OutputFormat[OutputFormat.Simple]) {
```

Comment:
Is it verified that `requestWordLevelTimestamps()` does internally set `OutputFormat.Detailed` for the condition here and a couple of other places (ref. https://dev.azure.com/speedme/Carbon/_workitems/edit/14947/)?